src/main/scala/org/zalando/nakadi/client/KlientImpl.scala

```diff
@@ -152,7 +152,7 @@ protected class KlientImpl(val endpoint: URI, val port: Int, val securedConnecti
   override def subscribeToTopic(topic: String, parameters: ListenParameters, listener: Listener, autoReconnect: Boolean): Future[Unit] = {
     getPartitions(topic).map{_ match {
       case Left(errorMessage) =>
-          throw new KlientException(s"a problem ocurred while subscribing to [topic=$topic, errorMessage=$errorMessage]")
+          throw new KlientException(s"a problem ocurred while subscribing to [topic=$topic]: $errorMessage")
```

Comment:
`errorMessage` is not part of what caused the problem, so I suggest to change the formatting. This is probably the only place where functionality is actually tampered with.